src/components/v2/AuthModal/WalletList/index.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Typography from '@mui/material/Typography';
+
+import { Connector } from 'clients/web3';
+import { VENUS_TERMS_OF_SERVICE_URL } from 'config';
+import { Icon } from '../../Icon';
+import { WALLETS, UPCOMING_WALLETS } from '../constants';
+import { useStyles } from './styles';
+
+export interface IWalletListProps {
+  onLogin: (connector: Connector) => void;
+}
+
+export const WalletList: React.FC<IWalletListProps> = ({ onLogin }) => {
+  const styles = useStyles();
+
+  return (
+    <div css={styles.container}>
+      {WALLETS.map(({ name, connector, Logo }) => (
+        <button
+          css={styles.getListItem({ isActionable: true })}
+          key={`wallet-${name}`}
+          type="button"
+          onClick={() => onLogin(connector)}
+        >
+          <Logo css={styles.walletLogo} />
+
+          <Typography css={styles.walletName} component="span">
+            {name}
+          </Typography>
+
+          <Icon name="chevronRight" css={[styles.chevronRightIcon]} />
+        </button>
+      ))}
+
+      <div css={styles.divider} />
+
+      {UPCOMING_WALLETS.map(({ name, Logo }) => (
+        <div css={styles.getListItem({ isActionable: false })} key={`upcoming-wallet-${name}`}>
+          <Logo css={styles.walletLogo} />
+
+          <Typography css={styles.walletName} component="span">
+            {name}
+          </Typography>
+
+          <Typography css={styles.comingSoonText} component="span">
```

Comment:
are these so different from out font styles that we can't use a variant?

Reply:
If we aren't using a variant, we could use the tag directly

Reply:
We do use a variant, in this case the default one: `body1`. However, this needs to be rendered as a `span` as `Typography` renders a `p` tag by default and this isn't a paragraph.

I know you don't like using `Typography`, but I believe that's how it is intended to be used. The only css applied to it via `styles.comingSoonText` is styling that isn't present in the variant (namely `color` and `text-align`).

Reply:
I don't have any problem using `Typography` and understand its purpose. I would wrap our default variants in their own components to make them less verbose.

Reply:
Yeah that works for me 👍